drivers/usb/device/usb_dc_kinetis.c

```diff
@@ -40,6 +40,13 @@ LOG_MODULE_REGISTER(usb_dc_kinetis);
 #define KINETIS_EP_NUMOF_MASK	0xf
 #define KINETIS_ADDR2IDX(addr)	((addr) & (KINETIS_EP_NUMOF_MASK))
 
+/*
+ * In some SoC USB0 base register is defined as USBFS0
+ */
+#if !defined(USB0) && defined(USBFS0)
+#define USB0				USBFS0
```

Comment:
Just a note, `#define USB0 (USB_Type *)DT_INST_REG_ADDR(0)` should work as well (not tested :smile:).

https://github.com/zephyrproject-rtos/zephyr/blob/b80a058e9043087f2a114310df48eab4638598b6/drivers/usb/udc/udc_kinetis.c#L1182